### PR TITLE
Overhaul GettingStarted: Wording etc.

### PR DESCRIPTION
**Pull request by Olf0**

### CODE REVIEW

**Comment by Olf0** [comment]
Please note https://github.com/sailfishos-chum/main/pull/48#issuecomment-1013873299

File: GettingStarted.md [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
-Follow the process in the README and contact 'lbt' for an account to 
-be set up.
+If you are not using the Sailfish&nbsp;OS OBS yet, you shall obtain an account there by contacting the user `lbt` t the IRC channel `#sailfishos`.
 
-When your account is set up, you will have a "Home Project".  This is where
-you can start creating packages.  The URL for your home project will be:
+When your account is set up, you will have a "Home Project".
+This is where you can start creating packages.
+The URL for your home project will be:<br />
 https://build.sailfishos.org/project/show/home:<username>
```
Comment (rinigus) [inline GettingStarted.md:12]:
```suggestion
https://build.sailfishos.org/project/show/home:username
```

**Review by rinigus — COMMENTED** [review]
LGTM. @piggz, please review

**Comment by piggz** [comment]
LGTM after suggested changes

File: GettingStarted.md (hunk 1 above)
Comment (Olf0) [reply GettingStarted.md:12]: That was not me.  See https://github.com/sailfishos-chum/main/blame/main/GettingStarted.md#L12

But you are right, that this is wrong: It must be escaped.
Will do.

Comment (Olf0) [reply GettingStarted.md]: Ugh, only `<username\>` works properly, `\<username\>` renders as "\\<username\>" at this location (i.e., in contrast to using `\<username\>` solitarily)!

**Pull request merged**